integration/main_test.go

```diff
@@ -318,8 +334,15 @@ func Randomize(str string) string {
 }
 
 // KillProcess kills the process by name. pkill is used.
-func KillProcess(name string) error {
-	output, err := exec.Command("pkill", "-x", fmt.Sprintf("^%s$", name)).CombinedOutput()
+func KillProcess(name string, signal syscall.Signal) error {
+	var command []string
+	if goruntime.GOOS == "windows" {
```

Comment:
I think we should not introduce windows change because it is commited after release 1.5.
Just focus on linux platform here.

https://github.com/containerd/containerd/commit/abf4de4985fdf8edf0220a055ff14b9c33c5547c

Reply:
ok